hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMDeletedBlockTransactionStatusManager.java

```diff
@@ -545,4 +638,93 @@ public void removeTransactionFromDNsCommitMap(List<Long> txIds) {
   public void removeTransactionFromDNsRetryCountMap(List<Long> txIds) {
     txIds.forEach(transactionToRetryCountMap::remove);
   }
+  
+  public void reinitialize(Table<String, ByteString> configTable) throws IOException {
+    // DB onFlush() will be called before reinitialization.
+    this.statefulConfigTable = configTable;
+    this.initDataDistributionData();
+  }
+
+  @VisibleForTesting
+  public Map<Long, TxBlockInfo> getTxSizeMap() {
+    return txSizeMap;
+  }
+
+  @VisibleForTesting
+  public static void setDisableDataDistributionForTest(boolean disabled) {
+    disableDataDistributionForTest = disabled;
+  }
+
+  @Nullable
+  public DeletedBlocksTransactionSummary getTransactionSummary() {
+    if (!VersionedDatanodeFeatures.isFinalized(HDDSLayoutFeature.STORAGE_SPACE_DISTRIBUTION)) {
```

Comment:
STORAGE_SPACE_DISTRIBUTION here also not required, by default will be 0 if not finalized, this is redundant check.